app/models/chargeable_field.rb

```diff
@@ -23,4 +23,14 @@ def self.seed_data
     fixture_file = File.join(FIXTURE_DIR, 'chargeable_fields.yml')
     File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
   end
+
+  def self.chargeable_cols_on_metric_rollup
+    existing_cols = MetricRollup.attribute_names
+    chargeable_cols = pluck(:metric) & existing_cols
+    chargeable_cols.map! { |x| VIRTUAL_COL_USES[x] || x }
+  end
+
+  VIRTUAL_COL_USES = {
```

Comment:
Do we?

Do we have any project level guidance?

The only guidance I regularly receive is Ruby Weekly newsletter. In some of their blogs somebody suggested to put private constants to the bottom.

If you can link to guidance that requires constants on top. I'll swiftly fix this with apology. :)